src/db.ts

```diff
@@ -0,0 +1,354 @@
+import _ from 'lodash';
+import { Client } from 'pg';
+import config from './config';
+
+const client = new Client(config.postgreSQL);
+
+// Table Names
+const CONVERSATIONS = 'conversations';
+const PARTICIPANTS = 'participants';
+const SESSIONS = 'sessions';
+const SEGMENTS = 'segments';
+const METRICS = 'metrics';
+
+// Create Queries
+const CREATE_CONVERSATIONS_TABLE_QUERY = `
+CREATE TABLE IF NOT EXISTS ${CONVERSATIONS}(
+    conversation_id                      VARCHAR(36) NOT NULL,
```

Comment:
GUIDs, and any guaranteed fixed-length strings, should use CHAR instead of VARCHAR. CHAR will use less data to store the same string.